File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/Skattekompensertbelop.java
package no.nav.domain.pensjon.regler.beregning2011;

import no.nav.domain.pensjon.regler.kode.FormelKodeCti;

import java.io.Serializable;

/**
 * @author Aasmund Nordstoga (Accenture) PK-5549
 */
public class Skattekompensertbelop implements Serializable {

    

    private double faktor;

    private FormelKodeCti formelKode;

    private double arsbelop;

    private Justertbelop justertbelop;

    private double tillegg;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/Skattekompensertbelop.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.enum.FormelKodeEnum
import java.io.Serializable

class Skattekompensertbelop : Serializable {
    var faktor = 0.0
    var formelKodeEnum: FormelKodeEnum? = null
    var arsbelop = 0.0
    var justertbelop: Justertbelop? = null
    var tillegg = 0.0

}